src/server/plugins/plugin.js

```diff
@@ -127,8 +127,10 @@ module.exports = class Plugin {
         server.exposeStaticDir(`/plugins/${id}/{path*}`, this.publicDir);
       }
 
-      this.status = kbnServer.status.createForPlugin(this);
-      server.expose('status', this.status);
+      if (this.externalInit !== _.noop) {
```

Comment:
well... because its never undefined and its always a function ? :)
this.externalInit will always be a function. 
- _.noop in case its not used ... and 
- a real function in case init was set.